### ISSUE TITLE
This project seems to depend on non-free software

### ISSUE THREAD

**Issue by muelli** [hieuvp/react-native-fingerprint-scanner#240]
there seems to be an unconditional dependency:

https://github.com/hieuvp/react-native-fingerprint-scanner/blob/v6.0.0/android/build.gradle#L52

which appears to pull in JAR files [without source code nor a licence](https://github.com/uccmawei/FingerprintIdentify/issues/100): https://github.com/uccmawei/FingerprintIdentify/tree/1.2.6/FingerprintIdentifyLib/libs

Can you make the dependency opt-in? So that the default behaviour is to depend on open source libraries, only. Thanks.

**Comment by BraveEvidence** [comment]
Check my video https://www.youtube.com/watch?v=DAZJ2JhDkNU. It will help you

**Comment by AlbertoMeQ** [comment]
I agree, also that library seems to be the case of the following logs:

```
.gradle/caches/transforms-3/2dbd6ab9e655cbe38e1c21660e2ed91c/transformed/jetified-fingerprintidentify-1.2.6-runtime.jar: R8: Expected stack map table for method with non-linear control flow. In later version of R8, the method may be assumed not reachable
```
